src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -161,8 +162,32 @@ export default class SpotTileModel extends ModelBase {
       this._log.warn(`Ignoring execute request as we can't trade with pricing and execution down`);
     }
   }
+  
+  _subscribeToClosePositionRequests(){
```

Comment:
I would move this logic to the execution service, given we don't seem to interact with the spot tile at all.